docs/client-side-usecases configuration.md

```diff
@@ -0,0 +1,105 @@
+last edit: 2016-11-02 11:13
+author: m4dz <code@m4dz.net>
+
+---
+
+
+Cozy MyAccounts Client Use-case
+===============================
+
+Abstract
+--------
+
+This document describes the context of using _use-cases_ in MyAccounts app, how they are related to the app and the stack, and how to customize them in hosting context.
+
+
+What are use-cases?
+-------------------
+
+A **use-case** is a way to sort and filter konnectors by an arbitrary denominator (e.g. _all konnectors that concerns billing_). It allow a user to browse available konnectors in a different way than browsing by categories. I allow the user to discover some konnectors by selecting first a _use-case_ (aka a scenario) that cover usages, and see what konnectors can be suggested.
+
+_MyAccounts_ can provides many _use-cases_, that can be configured, which mean a hoster can offer a different way to discover available konnectors (aka present different _use-cases_).
+
+
+How to define a use-case?
+-------------------------
+
+All _use-cases_ are declared in a manifest, built-in the client app, at `client/app/contexts/<context>/index.json`. Its architecture is defined ine the _Architecture of the manifest_ section below.
+
+The `context` value is the one defines in which context _MyAccounts_ app run. There must be a `cozy` context, which is the default one available. Later, all partners can defines their own _use-cases_ by adding a new context, with a dedicated manifest and locales (see _Localization_ section below about translations).
+
+Adding a _use-case_ simply mean adding a new object into the _use-cases_ array, that provides the following keys (see the _Architecture_ section below for a complete list and informations):
+
+- `slug`: `<String>` the slugname for the use-case
+- `konnectors`: `<Array[Obj]>` an array of objects defining the konnectors
+- `figure`: `<String>` the picture file that illustrate the use-case
+- `color`: `<Color>` a color object that defines the color of the use-case
+- `default`: `<Bool>` a `true|false` value that defines this use-case as the default one
```

Comment:
Are we ok that the default value for `default` property should be `false`, so we have only one use case containing this property (set to `true`) ?